backend/fms_core/serializers.py

```diff
@@ -379,11 +379,11 @@ class Meta:
                   'comment')
 
 class SampleExportSerializer(serializers.Serializer):
-    coordinate = serializers.CharField(read_only=True, source="coordinate.name")
+    coordinates = serializers.CharField(read_only=True, source="coordinate.name")
```

Comment:
so coordinate.name contains multiple coordinate?

Reply:
coordinate is the name of the model.  We typically use coordinates plural when speaking of a 2 dimensional location system as ours. The amount of coordinate (model) in coordinates is still 1.